pkgs/development/python-modules/textual-dev/default.nix

```
@@ -15,7 +15,7 @@
 
 buildPythonPackage rec {
   pname = "textual-dev";
-  version = "1.2.1";
+  version = "1.3.0";
   pyproject = true;
 
   disabled = pythonOlder "3.7";
```

```suggestion
  disabled = pythonOlder "3.8";
```
https://github.com/Textualize/textual-dev/commit/08f709f959abf9614f0d7541b4042b06a56d5b07